rls/src/build/external.rs

```diff
@@ -106,7 +106,7 @@ where
         let mut contents = String::new();
         file.read_to_string(&mut contents).map_err(|e| e.to_string())?;
 
```

Comment:
Similarly, from_read here should give some sweet perf improvements

Reply:
You'd think, but that's unfortunately [not the case in practice](https://github.com/serde-rs/json/issues/160#issuecomment-253446892) (the reason why I went with the string -> file flow in the first place but it seems to only affect reading)

Reply:
Wow =/